### DISCUSSION TITLE
Proposal: Drop support for MacOS 11 (Big Sur)

### DISCUSSION THREAD

**Discussion by brettle** [cyberbotics/webots#6624] [Ideas]
It is no longer supported by Apple and dropping support for it would probably allow us to [use homebrew for freetype](https://github.com/cyberbotics/webots/wiki/FreeType-compilation#mac) and possibly other dependencies. Thoughts?

**Comment by omichel** [comment]
That's already the case since #6580 for the upcoming version as shown in the documentation: https://cyberbotics.com/doc/guide/system-requirements?version=master

**Comment by brettle** [comment]
Thanks. My bad.
